omod/src/main/java/org/openmrs/module/aijar/fragment/controller/PatientStabilityFragmentController.java

```diff
@@ -192,4 +195,19 @@ public String getDateBefore(Date referenceDate, int noOfMoths,int noOfYears) {
         }
         return obsList;
     }
+
+    public Date getArtStartDate(Patient patient) {
+        List<Obs> list = Context.getObsService().getObservationsByPersonAndConcept(patient, Context.getConceptService().getConcept(99161));
+        if (list.size() <= 0) {
+            list = getObsListFromIdList("SELECT obs_id FROM obs where obs.person_id='" + patient.getPatientId() + "' AND obs.concept_id = 90315 AND obs.voided = false ORDER BY  obs.encounter_id DESC");
```

Comment:
@slubwama I do not think we should do this since it may be possible that the backlog for a patient has not been entered - so will give wrong data

Reply:
@ssmusoke does that mean that I only keep only for Summary Page??

Reply:
Yes correct, we only use this approach for the 106 as a proxy for the started ART in the quarter for new enrollments